src/databricks/sql/client.py

```diff
@@ -26,7 +22,7 @@ def __init__(
         self,
         server_hostname: str,
         http_path: str,
-        access_token: str,
+        oauth_persistence: OAuthPersistence = None,
```

Comment:
Can you add back `access_token: str = None` here? Consistent with the documentation below.

If you don't want to mention authorization related arguments here but rather keep them all in `**kwargs`, both `access_token` and `oauth_persistence` shouldn't be mentioned. Having one but not the other is inconsistent.

Reply:
thanks for the comment moved both to `**kwargs`